### DISCUSSION TITLE
Incompatible XDG Portal

### DISCUSSION THREAD

**Discussion by jrock2004** [hyprwm/Hyprland#2041] [Q&A] [answered]
When I start up hyprland I see a message about an incompatible xdg-portal. On my system I have hyprland and wayland gnome. If I check the xdg portals I have, this is what is installed

![image](https://user-images.githubusercontent.com/655716/231775346-0a49dc89-0b6a-4847-9564-baf3079c6873.png)

Could it be the first one I need to uninstall?

**Comment by vaxerski** [comment, marked as answer]
`-gnome` is incompatible, why not [read the wiki](https://wiki.hyprland.org/Useful-Utilities/Hyprland-desktop-portal/)

**Reply by jrock2004** [reply]
Sorry about that @vaxerski
